jr-annotation-support/pom.xml

```diff
@@ -76,6 +76,14 @@
         <groupId>de.jjohannes</groupId>
         <artifactId>gradle-module-metadata-maven-plugin</artifactId>
       </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+                <source>16</source>
```

Comment:
I assume this is just for testing -- Jackson 2.x is JDK 8, still.

Reply:
Thank you for taking a look at this draft PR. I appreciate the time and effort you invest in this open-source project :) The code in this PR still needs some refinement. I already expected that changing the JDK version is not wishful. This is just the first draft, where I focused on getting the functionality to work and gaining a better understanding of the library's internals.

Can I ask you a question about the RecordsHelper class? I've changed the visibility from default to public to use it in the AnnotationBasedIntrosepector class. I'm not sure if this change is appropriate. What do you think?

Reply:
@juulhobert Thanks! Apologies for slow follow up (was on vacation for a week).

So: yes, changing it to `public` is ok if (when) it is necessary.